packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -199,7 +203,7 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 						{ ! hideDesignTitle && (
 							<span className="design-picker__option-name">{ design.title }</span>
 						) }
-						{ ! isEnableThemePreviewScreen && isPremium && (
+						{ ! showBundledBadge && ! isEnableThemePreviewScreen && isPremium && (
```

Comment:
[nit] We're going to remove this block soon in a refactor PR as `isEnableThemePreviewScreen` is always `true`. Could we leave this as-is so that we don't have merge conflict? 😄